### ISSUE TITLE
Poll2 does not respect the lock file

### ISSUE THREAD

**Issue by tking53** [pixie16/paass#281]
I noticed when i was testing the MCAinPacman PR, that both @akeeler and i had a poll2 instance open at the same time. I believe this would cause some issues if we both tried to acquire data at the same time would it not?

**Comment by ksmith0** [comment]
Yes this would be a problem, did one of you delete the lock file before starting poll2? Improving the lock is on our list of things to do.
